xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/XWiki/SearchSuggestConfig.xml

```diff
@@ -1066,6 +1067,7 @@ qf=filename</query>
     </property>
     <property>
       <query>fq=type:ATTACHMENT
+fq=locale:*
```

Comment:
I'm less confident about that one. I think it means returning indexed content for all locales. But it's possible another property can be exploited to use the locale of the attachment itself (cc @mflorea)